### PR TITLE
Add print functionality point data modal

### PR DESCRIPTION
**Pull request by waracci**
## Description

When selecting a point on the point mapper the modal that appears used to have a print button
This was branched off https://github.com/OpenUpSA/wazimap-ng-ui/pull/431

## Screenshots

Screenshot from 2021-07-13 00-26-49

Screenshot from 2021-07-13 00-21-36

### Additional testing on browserstack
Screenshot from 2021-07-14 14-18-46
Screenshot from 2021-07-14 14-18-36
Screenshot from 2021-07-14 13-58-51

### CODE REVIEW

**Comment by waracci** [comment]
[point data print button is missing](https://trello.com/c/uYR53y2R/854-point-data-print-button-is-missing)

**Comment by adieyal** [comment]
@waracci - could you please resolve the conflict?

**Comment by waracci** [comment]
Converting to draft, merged with staging and something broke.

File: src/index.html [hunk 1]
```diff
@@ -3689,6 +3689,7 @@ <h4>Indicator chart title</h4>
       }
     });
   </script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jspdf/1.3.4/jspdf.debug.js"></script>
```
Comment (jbothma) [inline src/index.html:3692]: how is this added to the html? I don't see import-webflow-export import code to add it.

Comment (adieyal) [reply src/index.html:3692]: @waracci has this been resolved? I don't want to merge the code if we're adding something that will be clobbered when Matt exports again from webflow

**Comment by adieyal** [comment]
I'm getting an exception on linke 296 of point_data.js

```
Uncaught ReferenceError: facilityInfo is not defined
    PointData point_data.js:296
    PointData point_data.js:289
    jQuery 8
        dispatch
        handle
        add
        ke
        each
        each
        ke
        on
    PointData point_data.js:288
    PointData point_data.js:272
    jQuery 8
    PointData point_data.js:271
    PointData point_data.js:207
    PointData point_data.js:194
    fire Events.js:190
    _fireDOMEvent Map.js:1386
    _fireEvent Canvas.js:406
    _onClick Canvas.js:358
    handler DomEvent.js:79
    addOne DomEvent.js:114
    on DomEvent.js:34
    _initContainer Canvas.js:64
    onAdd Renderer.js:52
    onAdd Canvas.js:53
    _layerAdd Layer.js:110
    whenReady Map.js:1411
    addLayer Layer.js:172
    getRenderer Renderer.getRenderer.js:21
    beforeAdd Path.js:80
    addLayer Layer.js:169
    addLayer LayerGroup.js:47
    addLayer FeatureGroup.js:35
    PointData point_data.js:201
    checkIterate utils.js:202
```

Looks like facilityInfo is not defined.

**Comment by jbothma** [comment]
This is re-implementing rather than fixing the existing implementation

**Pull request closed without merging**